sys-devel/binutils/binutils-2.31.1.recipe

```diff
@@ -21,17 +21,18 @@ But they also include:
 The tools of this package can be used for cross-builds to \
 $effectiveTargetMachineTriple."
 HOMEPAGE="http://www.gnu.org/software/binutils"
-COPYRIGHT="1988-2016 Free Software Foundation, Inc."
+COPYRIGHT="1988-2018 Free Software Foundation, Inc."
 LICENSE="GNU GPL v3
 	GNU LGPL v3"
 REVISION="1"
-binutilsVersion="2.29"
+binutilsVersion="2.31.1"
 SOURCE_URI="https://ftp.gnu.org/gnu/binutils/binutils-$binutilsVersion.tar.xz"
-CHECKSUM_SHA256="0b871e271c4c620444f8264f72143b4d224aa305306d85dd77ab8dce785b1e85"
+CHECKSUM_SHA256="5d20086ecf5752cc7d9134246e9588fa201740d540f7eb84d795b1f7a93bca86"
 SOURCE_DIR="binutils-$binutilsVersion"
-PATCHES="binutils-$portVersion.patchset"
+#PATCHES="binutils-$portVersion.patchset"
```

Comment:
The patch does not apply nicely. It should either be fixed or remove. So, maybe:
```diff
-#PATCHES="binutils-$portVersion.patchset"
```